cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsPlatformResources.java

```diff
@@ -147,16 +146,28 @@
     @Value("#{'${cb.aws.disabled.instance.types:}'.split(',')}")
     private List<String> disabledInstanceTypes;
 
+    @Value("#{'${cb.aws.distrox.disabled.instance.types:}'.split(',')}")
+    private List<String> disabledDistroxInstanceTypes;
+
     private final Predicate<VmType> enabledInstanceTypeFilter = vmt -> disabledInstanceTypes.stream()
             .filter(it -> !it.isEmpty())
             .noneMatch(di -> vmt.value().startsWith(di));
 
+    private final Predicate<VmType> enabledDistroxInstanceTypeFilter = vmt -> disabledDistroxInstanceTypes.stream()
+            .filter(it -> !it.isEmpty())
+            .map(it -> it.toLowerCase().trim().replaceAll("\\s+", ""))
```

Comment:
could you please extract this replacement to a method with an example what's going to be replaced?